RocketChat/Rocket.Chat.ReactNative
# [IMPROVE] Accept user avatar url from external provider

## Proposed changes

Check if Avatar External Provider Url is enabled, thus providing a new URL format to display the user avatar

## Issue(s)	

Closes https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/3362

## How to test or reproduce


Our `mobile.rocket.chat` is using the `open.rocket.chat` as external provider. So, you must have an account on both servers with the same username

Then log in on `mobile.rocket.chat` with another user and send a DM to the previous one. Check if it has the same photo on both servers.


Observations: If you change the photo on `open.rocket` later started a DM on `mobile.rocket` with it, you probably will keep seeing the photo before the last update, this is because we are saving the photos in a cache. So, you need to go to Settings and Clear Local Cache.

## Screenshots

## Types of changes




## Checklist



## Further comments
